package.json

```diff
@@ -34,19 +34,43 @@
     "docs": "node scripts/docs.js"
   },
   "jest": {
-    "testURL": "http://localhost",
-    "setupFiles": [
-      "./jest.setup.js",
-      "jest-localstorage-mock"
-    ],
-    "setupFilesAfterEnv": [
-      "<rootDir>/packages/cozy-stack-client/src/__tests__/setup.js"
-    ],
-    "testMatch": [
-      "**/(*.)(spec|test).js?(x)"
-    ],
-    "modulePathIgnorePatterns": [
-      "<rootDir>/packages/.*/dist/"
+    "projects": [
+      {
+        "displayName": "Test JS files",
+        "testRegex": [
+          "(\\.|/)(test|spec)\\.js$"
+        ],
+        "testURL": "http://localhost",
+        "setupFiles": [
+          "./jest.setup.js",
+          "jest-localstorage-mock"
+        ],
+        "setupFilesAfterEnv": [
+          "<rootDir>/packages/cozy-stack-client/src/__tests__/setup.js"
+        ],
+        "modulePathIgnorePatterns": [
+          "<rootDir>/packages/.*/dist/"
+        ]
+        
+      },
+      {
+        "displayName": "Test JSX files",
+        "testMatch": [
+          "**/(*.)(spec|test).jsx"
+        ],
+        "testURL": "http://localhost",
+        "setupFiles": [
+          "./jest.setup.js",
+          "jest-localstorage-mock"
+        ],
+        "setupFilesAfterEnv": [
+          "<rootDir>/packages/cozy-stack-client/src/__tests__/setup.js"
+        ],
+        "browser": true,
+        "modulePathIgnorePatterns": [
+          "<rootDir>/packages/.*/dist/"
+        ]
+      }
```

Comment:
Most of the configs are shared, could they be regrouped in a commonConfig and then a `merge` could be used to only change what differs between the two ?

Reply:
https://github.com/cozy/cozy-client/pull/651/commits/6cab6ed73a8fec3b53a17e898ddc8d00c6c9ebdc done thanks :)